projects/1.19/assets/chipped/chipped/lang/zh_cn.json

```
@@ -0,0 +1,6983 @@
+  "block.chipped.small_coal_block_bricks": "煤炭小型砖",
+  "block.chipped.smooth_coal_block_column": "平滑煤炭柱石",
+  "block.chipped.smooth_inlayed_coal_block": "平滑嵌花煤炭块",
+  "block.chipped.smooth_ringed_coal_block": "平滑回纹煤炭块",
+  "block.chipped.smoothed_double_inlayed_coal_block": "平滑双重嵌花煤炭块",
+  "block.chipped.spider_coal_block_carving": "蜘蛛雕徽煤炭块",
+  "block.chipped.spiraled_coal_block": "螺旋纹煤炭块",
+  "block.chipped.stacked_coal_block_bricks": "叠层煤炭砖",
+  "block.chipped.thick_inlayed_coal_block": "厚沿嵌花煤炭块",
+  "block.chipped.tiled_coal_block": "方纹煤炭块",
+  "block.chipped.tiled_coal_block_column": "方纹煤炭柱石",
+  "block.chipped.tiled_bordered_coal_block": "方砖沿煤炭块",
+  "block.chipped.tiny_coal_block_bricks": "煤炭微型砖",
+  "block.chipped.tiny_brick_bordered_coal_block": "窄砖沿煤炭块",
+  "block.chipped.tiny_layered_coal_block_bricks": "密层煤炭砖",
+  "block.chipped.tiny_layered_coal_block_slabs": "细层煤炭砖",
+  "block.chipped.trodden_coal_block": "踏裂煤炭块",
+  "block.chipped.unamused_coal_block": "不悦雕徽煤炭块",
+  "block.chipped.vertical_cut_coal_block": "竖纹切制煤炭块",
+  "block.chipped.vertical_disordered_coal_block_bricks": "竖纹乱砌煤炭砖",
+  "block.chipped.weathered_coal_block": "风化煤炭块",
+
+  "block.chipped.crying_obsidian_bricks": "哭泣的黑曜石砖",
+  "block.chipped.crying_obsidian_mini_tiles": "哭泣的黑曜石小型瓦",
+  "block.chipped.crying_obsidian_pillar": "哭泣的黑曜石柱",
+  "block.chipped.crying_obsidian_pillar_top": "哭泣的黑曜石柱端",
+  "block.chipped.crying_obsidian_scales": "哭泣的黑曜石鳞块",
+  "block.chipped.angry_crying_obsidian": "愤怒雕徽哭泣的黑曜石",
+  "block.chipped.blank_crying_obsidian_carving": "空白式刻徽哭泣的黑曜石",
+  "block.chipped.bordered_crying_obsidian": "带沿网纹哭泣的黑曜石",
+  "block.chipped.brick_bordered_crying_obsidian": "砖沿哭泣的黑曜石",
+  "block.chipped.carved_crying_obsidian": "刻制哭泣的黑曜石",
+  "block.chipped.checkered_crying_obsidian_tiles": "棋盘格哭泣的黑曜石瓦",
+  "block.chipped.cobbled_crying_obsidian": "圆石纹哭泣的黑曜石",
+  "block.chipped.cracked_crying_obsidian_bricks": "裂纹哭泣的黑曜石砖",
+  "block.chipped.cracked_disordered_crying_obsidian_bricks": "裂纹乱砌哭泣的黑曜石砖",
+  "block.chipped.cracked_flat_crying_obsidian_tiles": "裂纹平整哭泣的黑曜石瓦",
+  "block.chipped.creeper_crying_obsidian_carving": "苦力怕徽哭泣的黑曜石",
+  "block.chipped.crying_crying_obsidian": "哭泣雕徽哭泣的黑曜石",
+  "block.chipped.curly_crying_obsidian_pillar": "螺卷纹哭泣的黑曜石柱",
+  "block.chipped.cut_blank_crying_obsidian": "空白式雕徽哭泣的黑曜石",
+  "block.chipped.cut_crying_obsidian_column": "切制哭泣的黑曜石柱石",
+  "block.chipped.duh_crying_obsidian": "疑惑雕徽哭泣的黑曜石",
+  "block.chipped.edged_crying_obsidian_bricks": "刻沿哭泣的黑曜石砖",
+  "block.chipped.engraved_crying_obsidian": "錾制哭泣的黑曜石",
+  "block.chipped.eroded_crying_obsidian": "风蚀哭泣的黑曜石",
+  "block.chipped.etched_crying_obsidian_bricks": "蚀刻哭泣的黑曜石砖",
+  "block.chipped.fine_crying_obsidian_pillar": "精细哭泣的黑曜石柱",
+  "block.chipped.flat_crying_obsidian_tiles": "平整哭泣的黑曜石瓦",
+  "block.chipped.glad_crying_obsidian": "欢愉雕徽哭泣的黑曜石",
+  "block.chipped.inlayed_crying_obsidian": "嵌花哭泣的黑曜石",
+  "block.chipped.inscribed_crying_obsidian": "纂刻哭泣的黑曜石",

```

根据其他行对于Inscribed的翻译，这里应该是`篆刻`
```suggestion
  "block.chipped.inscribed_crying_obsidian": "篆刻哭泣的黑曜石",
```

---

应该是没有剩余的了